src/components/app/Footer.tsx

```diff
@@ -29,7 +29,7 @@ export const Footer = (): JSX.Element => (
                     <FooterIconItem icon={faYoutube} href="https://www.youtube.com/c/FlyByWireSimulations"/>
                 </div>
                 <div className="text-center">
-                    <div className="grid grid-cols-2 w-5/6 sm:w-2/3 mx-auto">
+                    <div className="grid grid-cols-2 w-4/5 sm:w-2/3 mx-auto">
```

Comment:
```suggestion
                    <div className="grid grid-cols-2 w-3/5 mx-auto">
```
BEFORE:  
![image](https://user-images.githubusercontent.com/18389085/107681687-48130680-6c97-11eb-9edc-c162ee5de252.png)

AFTER: 
![image](https://user-images.githubusercontent.com/18389085/107681717-53663200-6c97-11eb-8bb6-e0cd4612349c.png)


All views are on small.

I recommend you change to this.